validator/client/service.go

```diff
@@ -250,3 +253,26 @@ func ConstructDialOptions(
 
 	return dialOpts
 }
+
+// fetch Graffiti from a file or flag.
+func fetchGraffiti(flag string) [][]byte {
+	if flag == "" {
+		// No graffiti.
+		return make([][]byte, 1)
+	}
+
+	data, err := ioutil.ReadFile(flag)
+	if err != nil {
+		// Single graffiti.
```

Comment:
I think this is might be poor assumption. Any error at all means the user intended for there to be a single graffiti of the flag value? What about a file permissions error?

Reply:
Agree it needs better error checking.

Reply:
How about putting this into a separate option "--graffiti-file" instead of overloading the existing one?